sql/2022/pwa/assetlink_usage.sql

```diff
@@ -0,0 +1,61 @@
+#standardSQL
+# assetlink usage
+
+SELECT
+  'PWA sites' AS type,
+  _TABLE_SUFFIX AS client,
+  COUNT(0) AS freq,
+  total,
+  COUNT(0) / total AS pct
+FROM
+  `httparchive.pages.2022_07_01_*`
```

Comment:
We're using June data not July for Web Almanac 2022.

Can you update all the queries to be June and rerun?

Reply:
That's fine. Are you suggesting taking the 2022_06_09_* tables? Or 2022_06_01_*?

Reply:
`2022_06_01_*`

Reply:
Updated